apps/mdn/mdn-aws/infra/mdn-cdn/cloudfront_attachments/cloudfront_attachments.tf

```diff
@@ -0,0 +1,73 @@
+variable "distribution_name" {}
+variable "comment" {}
+variable "domain_name" {}
+variable "acm_cert_arn" {}
+
+variable "aliases" {
+  type = "list"
+}
+
+resource "aws_cloudfront_distribution" "mdn-attachments-cf-dist" {
+  aliases         = "${var.aliases}"
+  comment         = "${var.comment}"
+  enabled         = true
+  http_version    = "http2"
+  is_ipv6_enabled = false
+  price_class     = "PriceClass_All"
+
+  custom_error_response {
+    error_caching_min_ttl = 300
+    error_code            = 404
+    response_page_path    = "/404.html"
```

Comment:
This page doesn't exist:

https://mdn.mozillademos.org/404.html

I think the line should be removed.

Reply:
:+1: